dbaccess/subnetwork.go

```diff
@@ -0,0 +1,50 @@
+package dbaccess
+
+import (
+	"github.com/kaspanet/kaspad/database2"
+	"github.com/kaspanet/kaspad/util/subnetworkid"
+)
+
+var subnetworkBucket = database2.MakeBucket([]byte("subnetworks"))
+
+// FetchSubnetworkData returns the subnetwork data by its ID.
+func FetchSubnetworkData(context Context, subnetworkID *subnetworkid.SubnetworkID) ([]byte, error) {
+	accessor, err := context.accessor()
+	if err != nil {
+		return nil, err
+	}
+
+	key := subnetworkBucket.Key(subnetworkID[:])
+	return accessor.Get(key)
+}
+
+// RegisterSubnetwork stores mappings from ID of the subnetwork to the subnetwork data.
+func RegisterSubnetwork(context Context, subnetworkID *subnetworkid.SubnetworkID, subnetworkData []byte) error {
```

Comment:
"Register" is a bit out of scope for dbaccess. Please rename to StoreSubnetwork.